lib/src/source/hosted.dart

```diff
@@ -225,6 +268,26 @@ class _VersionInfo {
   _VersionInfo(this.pubspec, this.archiveUrl, this.status);
 }
 
+class _HostedDescription {
```

Comment:
Can we give this a documentation comment? Maybe say it's the value of `PackageRef.description` for `HostedSource`.

I guess this is replacing the `Pair<String, Uri>` we had before. That's probably good :D